R/utils-time.R

```diff
@@ -34,6 +35,12 @@ flexibleTimeParse <- function(x, timezone, shiftTimeToNoon = TRUE) {
     time[which.still.NA] <- time[which.still.NA] + hours(12)
   }
   
+  #Apply UTC offset
+  if (applyOffset){
```

Comment:
I was under the impression that `parse_date_time(x,format_str, tz=timezone,quiet=TRUE)` would parse the times into the timezone specified in the `timezone` argument. So even if the times were provided in UTC if a CST timezone was provided they would be parsed to CST and have the offset applied there automatically. 

Is that not doing that?

Reply:
I should probably rename this variable, since it's not really "apply offset" but more "pull out offset for display"

Reply:
Ahh I see what this is doing now. Gotcha, okay!

Wondering if perhaps that might be better to have in a separate function like `formatTimeForDisplayWithoutOffset` or something?